applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -101,6 +101,10 @@ For information about how to upgrade the device firmware using a PC or a smartph
 .. note::
     Currently the DFU over Bluetooth LE SMP and Matter OTA are not available for the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
 
+There is an additional functionality, which allows the Matter bridge device to work as a smart plug device.
+Thanks to it, you can integrate the Matter bridge functionality in a real end product, like smart plug, and do not have to make a bridge device work as a standalone device, which is much less attractive option.
+It is an optional feature, and can be enabled by passing accurate comman, see the :ref:`configuration_matter_bridge_functionality` section.
```

Comment:
I would add one extra newline before this sentence. I will be better exposed on the webpage.